tools/cli/commands/generate.js

```diff
@@ -378,7 +378,19 @@ async function generatePluginFromStarter( projDir, answers ) {
 	files = files.split( '\n' ).map( str => str.replace( 'projects/plugins/starter-plugin', '' ) );
 	files.forEach( file => {
 		if ( file && ! file.startsWith( 'changelog/' ) ) {
-			copyFile( path.join( projDir, file ), path.join( starterDir, file ) );
+			if ( file === 'tests/php/Jetpack_Starter_Plugin_Test.php' ) {
+				copyFile(
+					path.join(
+						projDir,
+						'tests/php/' + transformToPhpClassName( answers.name, false ) + '_Test.php'
```

Comment:
This seems to not be quite right. When I tried it for a plugin named "foobar", the file was named "Foobar_Test.php" but the class inside is "Jetpack_Foobar_Test".

Reply:
3349856eb0